site/en/docs/webstore/discovery/index.md

```diff
@@ -39,8 +42,10 @@ but there is no set checklist developers can follow to guarantee being featured.
 pay to be featured on the home page. The best thing a developer can do to increase their chances of
 being featured on the home page is to build a useful, high-quality extension that is a joy to use.
 
-{% Img src="image/WlD8wC6g8khYWPJUsQceQkhXSlv1/O4HjVbAcyIvYCxmCRB9H.png", width="800", height="571",
-   class="screenshot", alt="Screenshot of the Chrome Web Store home page." %}
+<figure data-size="full">
```

Comment:
Can we update this with a screenshot taken in a fresh browser on the real URL? I've sent you one in chat.